amp-camp/src/server/server.js

```diff
@@ -50,19 +57,48 @@ const listener = app.listen(port, () => {
 
 /** HANDLERS FOR STATIC PAGES AND STATIC FILES **/
 
+// Optimize and cache HTML responses
+app.use(function(req, res, next) {
+  if (req.method != 'GET' || !req.accepts('text/html')) {
+    return next();
+  }
+  // Set max-age to 1 min
+  res.set('Cache-Control', 'max-age=60');
```

Comment:
I'm not that familiar with the technique... can you explain a bit why the max-age is set to 1 min?

We do also want to use the max-age to tell the AMP Cache how long to cache certain page for. (Not that we're doing that yet.)

Reply:
Yeah, I've noticed that you're not setting max-age at all. 1 min is just a guess.

Reply:
I'm told that it should be about a week for product detail pages. We'll want a shorter time for the product listing page if I ever get around to making that hybrid-rendered. For now, would you mind doing a week? And putting that value in a constant up top somewhere?

Thanks for noticing this glaring gap... so many things we haven't gotten to...